clients/drcachesim/tests/trace_interval_analysis_unit_tests.cpp

```diff
@@ -85,15 +92,42 @@ class test_stream_t : public scheduler_t::stream_t {
     scheduler_t::input_ordinal_t
     get_input_stream_ordinal() override
     {
-        return 0;
+        assert(at_ > 0);
+        // All TIDs form a separate input stream.
```

Comment:
I don't understand this comment.  Do you mean, the tid values must be 0-based ordinals?  That should be in the top-level comment for the class?

Reply:
This comment doesn't say that. It only says that there are multiple input streams in the single output stream presented by the test_stream_t.

>  the tid values must be 0-based ordinals

I did not ensure or expect that currently. Made changes to map tid to a 0-based ordinal.

Reply:
You mean "Each TID forms a separate input stream"?  The "All" sounds like they all combine into one stream.

Reply:
Done.